gtars-core/src/models/region.rs

```diff
@@ -41,3 +41,22 @@ impl Display for Region {
         write!(f, "{}", self.as_string())
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct ChromosomeStats {
+    pub chromosome: String,
+    pub count: u32, // number of regions
+    pub start: u32,
+    pub end: u32,
+    pub minimum: u32,
+    pub maximum: u32,
+    pub mean: f64,
+    pub median: f64,
```

Comment:
Can we come up with some more descriptive names for these maybe? Right now its not immediately clear to me what these are.